src/server/views/repeating-summary.html

```diff
@@ -11,7 +11,7 @@
       <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l">{{customText.separatePageTitle or pageTitle }}</h1>
         {{ govukSummaryList(details) }}
-        <form action="?view=summary"  method="post" enctype="multipart/form-data">
+        <form action="?view=summary"  method="post" enctype="multipart/form-data" novalidate>
```

Comment:
Tiny extra space from the previous change

```suggestion
        <form action="?view=summary" method="post" enctype="multipart/form-data" novalidate>
```

Reply:
Resolved with 4526d79ca3ba089732f7db668abb36d8bff5e2a2

(FWIW, this file, and none of the "repeating" stuff is actually used and can probably be deleted at some point)